Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.controller.ts

```diff
@@ -89,14 +90,14 @@
                     (today > moment(itSystemUsage.concluded, Kitos.Constants.DateFormat.DanishDateFormat).startOf("day") && itSystemUsage.expirationDate == null) ||
                     (today < moment(itSystemUsage.expirationDate, Kitos.Constants.DateFormat.DanishDateFormat).endOf("day") && itSystemUsage.concluded == null) ||
                     (itSystemUsage.expirationDate == null && itSystemUsage.concluded == null)) {
-                    itSystemUsage.isActive = true;
+                    itSystemUsage.activeAccordingToValidityPeriod = true;
```

Comment:
the calculation whether it is active or not does not belong here. Once you get to the "status" summary task, see my solution for contracts.